bin/format_exercises

```diff
@@ -1,17 +1,36 @@
 #!/bin/sh
 # Format existing exercises using rustfmt
+set -e
 
 RUST_TRACK_REPO_PATH=$(cd "$(dirname "$0")/.." && pwd)
 
-EXERCISES_PATH="${RUST_TRACK_REPO_PATH}/exercises/"
+# traverse either concept or practice exercise
+# directory and format Rust files
+format_exercises() {
+    EXERCISES_PATH="${RUST_TRACK_REPO_PATH}/exercises/$1"
+    source_file_name="$2"
+    for exercise_dir in "${EXERCISES_PATH}"/*; do
+        (
+        cd "$exercise_dir"
+	    allowed_file="$exercise_dir"/.meta/ALLOWED_TO_NOT_COMPILE
+	    if [ -f "$allowed_file" ]; then
+	        exercise=$(basename "$exercise_dir")
```

Comment:
Indentation got weird here.

Reply:
Nice catch! My vim indentation messes this up despite editorconfig. Inspires me to enforce soon in CI!